### ISSUE TITLE
"Possessive Apostrophe" in German causes trouble in the localisation

### ISSUE THREAD

**Issue by FreezyLemon** [ppy/osu-web#3198]
Granted, this is pretty low priority, but what they call the [possessive apostrophe](https://en.wikipedia.org/wiki/Apostrophe#Possessive_apostrophe) in english is not quite as straightforward in german.. [this](https://german.stackexchange.com/questions/2550/does-german-language-have-possessive-apostrophe) could help.

TL;DR:
in German the following is correct:
`FreezyLemons Posts` (no apostrophe)
*but*
`Lucas' Posts` and `Max' Posts` (if the word ends on an "s" sound, the apostrophe is added again, but at the end)

The current localisation only allows either `'s` which is wrong, or just `s` which creates things like `Lucass Posts`.

**Comment by nanaya** [comment]
Maybe reword it to German equivalent of `Posts by FreezyLemon` instead?

**Comment by nanaya** [comment]
I don't think this can be fixed anytime soon. And as mentioned earlier there's probably other ways to reword it.
